packages/cli/src/metrics/prometheus-metrics.service.ts

```diff
@@ -65,6 +65,22 @@ export class PrometheusMetricsService {
 		}
 	}
 
+	enableLabels(labels: string[]) {
+		for (const label of labels as Array<
+			keyof (typeof PrometheusMetricsService)['prototype']['includes']['labels']
```

Comment:
Can we please instead create a predefined type for `includes` outside this class, and use it on line 27, 69, and 77.